docs/launch-in-aws.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Launch CyberArk Conjur In The Cloud
+layout: page
+section: get-started
+---
+
+You can easily launch a private, production-ready Conjur server in the
+cloud using Conjur's official template for Amazon CloudFormation.
+
+## Prerequisites
+
+First, you'll need an Amazon Web Services account. If you already have
```

Comment:
Do we really think people will be here without an AWS account already?

Reply:
Approaching it this way makes it more friendly. Probably nobody with "devops" in their job title will be without an AWS account, but people from more traditional ops, juniors without much ops experience, people from areas of the world where AWS isn't as ubiquitous might appreciate the link.